src/docs/sphinx/dev_guide/component_org.rst

```diff
@@ -214,20 +214,20 @@ All components should follow this format to describe the library information.
 Component docs directory
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A component `docs` directory contains a `CMakeLists.txt` file that uses
-the CMake `add_subdirectory` command to add `sphinx` and `doxygen` 
+A component `docs` directory contains a ``CMakeLists.txt`` file that uses
+the CMake ``add_subdirectory`` command to add `sphinx` and `doxygen` 
 subdirectories to the build configuration. These should be guarded to prevent
```

Comment:
Maybe add a short code snippet example for how to guard the add_subdirectory stuff?

Reply:
well this isn't even correct at all.  i wasnt looking at the content just the formatting.

I'll rework this